src/libsyntax/diagnostic_list.rs

```diff
@@ -16,31 +16,394 @@
 register_long_diagnostics! {
 
 E0533: r##"
+The export_name attribute was malformed.
```

Comment:
I wondered about it. I decidec to change it if someone else thought the same so I will.